edenia/antelope-tools
# Chore/colors adjustments

### Adjust colors in charts to the theme

### What does this PR do?

- Manually set a list of colors for dark and light mode
- Resolve #1344
- Allow to click the producer row to open the profile
- The icon to change the mode was inverted
- Fix accessibility errors on block producers page and the sidebar
- Adjust stroke width in the rewards distribution page

### Steps to test

1. Run the project locally
2. Go to cpu benchmark, block distribution, rewards distribution and homepage 
3. Check the charts in both theme modes
4. Go to block producers page
5. Try clicking on the row to open a BP profile.

### Screenshots

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/4018ee6c-7f68-4cc4-a9a4-04cc46b0f966)

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/45460710-19f8-45c8-80e1-b32f19109fd6)
